target/arduino/export.bat

```diff
@@ -5,106 +5,101 @@
 
 :: Options are: 
 ::  -h  --help    Help and information.
-::  -c  --create  ^<directory^> Create Arduino library at specified location.
-::                              When a directory is found with equal name the 
-::                              user is informed and the script is aborted.
-::  -r  --replace ^<directory^> Replace Arduino library at specified location erasing existing files.
-::                              When a directory is found with equal name the 
-::                              user is requested to confirm deletion of the directory.
 
 :: \name Remko Welling (pe1mew@gmail.com)
 
 SET option=%1
 SET source_directory=%~dp0
-SET target_directory=%2
+
+:: Select operation: "-" is command, else run
+IF  "%option:~0,1%"=="-" (
+   GOTO :CASE
+) ELSE (
+   GOTO :RUN
+)
+GOTO :EOF
+
+:CASE
+:: Command handeling
 
 2>NUL CALL :CASE_%option% 
 
 IF ERRORLEVEL 1 CALL :DEFAULT_CASE
 
 EXIT /B
 
-:CASE_-r
-:CASE_--replace
-:: remove existing folder structure and copy new library at specified location.
-   IF EXIST "%target_directory:"=%\BasicMAC" (
-      GOTO :DO_REMOVE
-   ) ELSE (
-      ECHO [ERROR] Destination directory "%target_directory:"=%\BasicMAC" does not exist. Please correct destination name.
-   )
-GOTO :EOF
-
-:CASE_-c
-:CASE_--create
-:: create the directory and copy files to it, aborting if it already exists.
-   IF EXIST "%target_directory:"=%\BasicMAC" (
-      ECHO [ERROR] Destination directory "%target_directory:"=%\BasicMAC" does exist. Please correct destination name.
-   ) ELSE (
-      GOTO :DO_CREATE
-   )
-GOTO :EOF
-
 :CASE_-h
 :CASE_--help
    ECHO Create an Arduino-compatible library in the given directory, overwriting existing files.
    ECHO -h  --help    This information.
-   ECHO -c  --create  ^<directory^> Create Arduino library at specified location.
-   ECHO -r  --replace ^<directory^> Replace Arduino library at specified location erasing existing files.
 GOTO :EOF
 
 :DEFAULT_CASE
-   ECHO Unknown command or abort. Type: export.bat -h or --help for options.
+   ECHO Unknown command. Type: export.bat -h or --help for options.
+GOTO :EOF
+
+:RUN
+:: Normal operation.
```

Comment:
I would suggest doing something like `SET target_directory=%1` here, so you can just keep using `target_directory` below (using `option` as the directory name seems a bit confusing IMHO).

Reply:
It's a coding style thing I do support. its done.